setup.py

```diff
@@ -1767,7 +1767,7 @@ def detect_expat_elementtree(self):
                 ('XML_POOR_ENTROPY', '1'),
             ]
             extra_compile_args = []
-            expat_lib = []
+            expat_lib = ['m']
```

Comment:
Would you mind to add a comment mentioning "bpo-44394: " and explaining that libexpact uses isnan() of math.h?

Reply:
Done